ReactiveCocoa/Swift/Signal.swift

```diff
@@ -1277,3 +1277,35 @@ extension SignalType where Error == NoError {
 		}
 	}
 }
+
+/// Used with the `debug` operator to customize 
+/// the `Event` string.
+///
+/// This is useful, when you want to do more than
+/// just printing to the standard output.
+public protocol EventLogger {
```

Comment:
I'm not sure I see utility of `EventLogger` when you could just as easily write a `Event<Value, Error> -> ()` with `on(event:)`.